### PR TITLE
rhv - 46139 additions to VM Portal UI descriptions

### PR DESCRIPTION
**Pull request by emarcusRH**
https://issues.redhat.com/browse/RHV-46139

Changes proposed in this pull request:

-  add descriptions for pooled VMs
-  add suspended to Run option description 

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @sgratch )

### CODE REVIEW

**Comment by sgratch** [comment]
@emarcusRH  My comments were sent via mail.

File: source/documentation/doc-Introduction_to_the_VM_Portal/topics/Graphical_User_Interface_elements.adoc [hunk 1]
```diff
@@ -50,12 +50,17 @@ The number of currently displayed virtual machines or pool entities and the tota
 
 .Virtual machines pane
 
-The virtual machines pane displays virtual machine cards with the icon, operating system, name, status, and management options for each virtual machine or pooled virtual machine.
+The virtual machines pane displays virtual machine cards with the icon, operating system, name, status, and management options for each virtual machine or virtual machine pool.
+
+* Virtual machine pool cards -  include a colored label of the pool name, an icon, operating system label, name, Allocated VMs number, the number of Available VMs from this Pool, and a button that allows the user to *Take a virtual Machine*.
++
+Virtual machine pools are always displayed for Administrator level users, and for non-admin users that have pooled virtual machines assigned to them.
```
Comment (sgratch) [inline source/documentation/doc-Introduction_to_the_VM_Portal/topics/Graphical_User_Interface_elements.adoc:57]:
```suggestion
Virtual machine pools are always displayed for Administrator level users, and for non-admin users that have permissions to view those pools and take pooled virtual machines out of those pool
```

**Review by sgratch — COMMENTED** [review]
Looks good except one fix

**Comment by sgratch** [comment]
@emarcusRH in addition, we changed the web-ui layout in RHV 4.4.z Service Pack 1 Batch #1 [ovirt-4.5.1] to use Patternfly 4, so I suggest using a new screenshot instead of https://github.com/oVirt/ovirt-site/commit/e31660d090d95d88fe4380c0b65d80e8a7d45556#diff-94c2743913ef7f5baf4cd727fb1842b2330e79186d9f680c2838267754dd28d6

**Comment by emarcusRH** [comment]
> @emarcusRH in addition, we changed the web-ui layout in RHV 4.4.z Service Pack 1 Batch #1 [ovirt-4.5.1] to use Patternfly 4, so I suggest using a new screenshot instead of [e31660d#diff-94c2743913ef7f5baf4cd727fb1842b2330e79186d9f680c2838267754dd28d6](https://github.com/oVirt/ovirt-site/commit/e31660d090d95d88fe4380c0b65d80e8a7d45556#diff-94c2743913ef7f5baf4cd727fb1842b2330e79186d9f680c2838267754dd28d6)

@sgratch can you provide me with new screen captures? Or a VM Portal login that has the updates?

**Comment by emarcusRH** [comment]
@sgratch HI Sharon, 
I have updated the screen image. Please give a review approval so that I can finalize this item.

**Review by sgratch — COMMENTED** [review]
> HI Sharon,
> I have updated the screen image. Please give a review approval so that I can finalize this item.

LGTM. The only thing I would consider is enlarging/zoom out the window's dialog to avoid cutting the filter fields due to lack of space, so that it would look like this:
image

Instead of current image filter which looks like this ("filter by name" field is collapsed):
image

**Review by sgratch — APPROVED** [review]
LGTM

**Review by sgratch — APPROVED** [review]

**Review by nunzy1 — APPROVED** [review]

**Pull request merged**